_posts/2020-06-21-one-stop-guide-to-lombok.md

```diff
@@ -0,0 +1,337 @@
+---
+title: "One-stop Guide to Lombok"
+date: 2020-06-21 10:00 +0500
+modified: 2020-06-21 10:15 +0500
+---
+
+# What is it
+Lombok is a Java library which makes Java less chatty whilst making your code more readable. Java is great language but when you use its features you have to write a lot of boilerplate code.
```

Comment:
All text is in "you-speak", but should be in "we-speak". Please go through the text and replace all "you" with "we".

Reply:
How does Lombok add code? Add a sentence or two about the annotation processor and how it works.